analyzer/src/funTest/assets/projects/synthetic/yarn-workspaces/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "workspace",
+  "version": "1.0.0",
+  "description": "",
```

Comment:
Please add a description similar to the one we use in the NPM test projects.

Reply:
Done.